src/protos/workflow.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package pb;
+
+import "processing.proto";
+
+service Workflow {
+    rpc WorkflowGet (WorkflowGetRequest) returns (WorkflowGetResponse) {}
+}
+
+// rpc ApproveTransfer
+
+message WorkflowGetRequest {
+    string id = 1;
+}
+
+message WorkflowGetResponse {
+    WorkflowData workflow = 1;
+    string error = 2;
+}
+
+// Other message types
+
+message WorkflowData {
+    repeated WatchedDirectory watchedDirectories = 1;
+    map<string,Chain> chains = 2;
+    map<string,Link> links = 3;
+}
+
+message WatchedDirectory {
+    enum WatchedDirectoryType {
+        UNKNOWN = 0;
+        TRANSFER = 1;
+        SIP = 2;
+        DIP = 3;
+    }
+    WatchedDirectoryType type = 1;
+    string path = 2;
+    bool onlyDirs = 3;
+    string chainId = 4;
+}
+
+message Chain {
+    map<string, string> description = 1;
+    string linkId = 2;
+    string id = 3;
+}
+
+message Link {
+    message LinkExitCode {
+        uint32 code = 1;
+        Job.Status jobStatus = 2;
+        string linkId = 3;
+    }
+    message LinkConfig {
+        message Standard {
+            string execute = 1;
+            string arguments = 2;
+            string filterSubdir = 3;
+            string filterFileStart = 4;
```

Comment:
I think it makes sense to get rid of it. I've done the same with other unused fields IIRC. I will double check before but sure!